docs/source/api/core/propagators.rst

```diff
@@ -0,0 +1,4 @@
+Propagators Module
+==================
+.. automodule:: poliastro.core.propagation
```

Comment:
Only one of the methods is documented (similar problem as above), perhaps I would add a one-line docstring to the other ones?